src/daemon/abrt-action-save-package-data.c

```diff
@@ -274,6 +274,13 @@ static int SavePackageDescriptionToDebugDump(const char *dump_dir_name, const ch
         goto ret; /* return 1 (failure) */
     }
 
+    /* Check well-known interpreter names */
```

Comment:
Please, help me to understand how you check `well-known` interpreter names here.